src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {
+  useState,
+  ReactElement,
+  FC,
+  SyntheticEvent,
+  Children,
+  cloneElement,
+} from 'react';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: any | null;
```

Comment:
derp! just read the component readme

> This components takes an arbitrary amount of `ExpansionPanel` components as children

I'll go ahead and set that as the children type

Reply:
stuck with more generic ReactNode